### PR TITLE
Update ServerSettings.md

### PR DESCRIPTION
**Pull request by Goro2030**
Added instructions to check which stores, users are running on the server

### CODE REVIEW

File: docs/FAQ/ServerSettings.md [hunk 1]
```diff
@@ -64,6 +64,26 @@ yes
 
 Or see this [LunaNode example](https://github.com/JeffVandrewJr/patron/blob/master/SSH.md) with PuTTY.
 
+### How to see, as an admin, what's running on my server?
+
+You need to SSH into your BTCPay server, and run a single line to see the list of APPS in the system:
```
Comment (Bangalisch) [inline docs/FAQ/ServerSettings.md:69]:
```suggestion
You need to SSH into your BTCPay Server, and run a single line to see the list of `apps` in the system:
```

File: docs/FAQ/ServerSettings.md [hunk 2]
```diff
@@ -64,6 +64,26 @@ yes
...
+
+```
+docker exec -ti $(docker ps -a -q -f "name=postgres_1") psql -U postgres -d btcpayservermainnet -c 'select "Name" from "Apps";'
+```
+
+This to see the list of Stores and it's WebSites:
```
Comment (Bangalisch) [inline docs/FAQ/ServerSettings.md:75]:
```suggestion
This to see the list of `stores` and their websites:
```

File: docs/FAQ/ServerSettings.md [hunk 3]
```diff
@@ -64,6 +64,26 @@ yes
 
 Or see this [LunaNode example](https://github.com/JeffVandrewJr/patron/blob/master/SSH.md) with PuTTY.
 
+### How to see, as an admin, what's running on my server?
```
Comment (Bangalisch) [inline docs/FAQ/ServerSettings.md:67]:
```suggestion
### How to see, as an admin, what's running on my BTCPay Server?
```

File: docs/FAQ/ServerSettings.md [hunk 4]
```diff
@@ -64,6 +64,26 @@ yes
...
+docker exec -ti $(docker ps -a -q -f "name=postgres_1") psql -U postgres -d btcpayservermainnet -c 'select "StoreName","StoreWebsite" from "Stores";'
+```
+
+You can also run this to see the users list:
+
+```
+docker exec -ti $(docker ps -a -q -f "name=postgres_1") psql -U postgres -d btcpayservermainnet -c 'select "Id", "Email" from "AspNetUsers";'
+```
+
 ### How can I see my BTCPay version?
```
Comment (Bangalisch) [inline docs/FAQ/ServerSettings.md:87]:
```suggestion
### How can I see my BTCPay Server version?
```

File: docs/FAQ/ServerSettings.md [hunk 5]
```diff
@@ -64,6 +64,26 @@ yes
...
 
 You can see your BTCPay version in the **bottom right of the page footer** when you're logged in as a server admin.
```
Comment (Bangalisch) [inline docs/FAQ/ServerSettings.md:89]:
```suggestion
You can see your BTCPay Server version in the **bottom right of the page footer** when you're logged in as a server admin.
```

**Review by Bangalisch — COMMENTED** [review]
Hey! looks good! 
Thanks for the updates.
Left a review on a couple of "typos" :) 

Cheers!

**Comment by Goro2030** [comment]
> Hey! looks good! Thanks for the updates. Left a review on a couple of "typos" :)
> 
> Cheers!

Thanks for the corrections! All approved now.

**Comment by Goro2030** [comment]
@pavlenex , can you approve this one?

**Review by dennisreimann — APPROVED** [review]

**Comment by dennisreimann** [comment]
Thanks for the contribution!

**Pull request merged**